server/odc-service/src/main/java/com/oceanbase/odc/service/session/OdcStatementCallBack.java

```diff
@@ -390,7 +407,7 @@ private String getTraceIdAndAndSetStage(Statement statement, TraceWatch traceWat
             setExecuteTraceStage(traceWatch, executeDetails, stopWatch);
             return executeDetails.getTraceId();
         } catch (Exception ex) {
-            log.warn("Query sql execute details failed, reason={}", ex.getMessage());
+            log.warn("Query sql execute details failed.", ex);
```

Comment:
there is no need to print all err stacks

Reply:
done

Reply:
done